packages/neuvector/package.yaml

```diff
@@ -1,6 +1,5 @@
-url: https://neuvector.github.io/neuvector-helm/core-2.2.2.tgz
-version: 101.0.0
-doNotRelease: true
+url: https://neuvector.github.io/neuvector-helm/core-2.2.3.tgz
+version: 100.0.2
```

Comment:
dev-v2.7 should be incrementing the minor version by 1. See chart versioning guidelines in https://github.com/rancher/charts#versioning-charts.

The version in this PR should be `100.1.0`, which will come out as `100.1.0+up2.2.3` in `make charts`.

Reply:
I will update the version to 100.1.0

Reply:
Apologies for this! Should have been 101.0.0 as listed in the versioning guidelines I linked (but incorrectly quoted 😓)